specification/resources/resource-manager/readme.md

```diff
@@ -74,9 +74,18 @@ tag: package-changes-2022-05
 ```
 
 ``` yaml $(package-snapshots)
-tag: package-snapshots-2022-11
+tag: package-2022-12
```

Comment:
Sorry if it's a dumb question, but why is this changing?

Reply:
Correctly pointed, this shouldn't change.

Reply:
Fixed